### ISSUE TITLE
express-nedb-rest and NeDB timestampData option

### ISSUE THREAD

**Issue by CharlesNepote** [bi-tm/express-nedb-rest#6]
I'm using the `timestampData` option of NeDB:
```javascript
// ---------- create  NEDB datastore
// timestamData creates a "createdAt" field with ISO 8601 timestamp: 2017-04-06T08:39:44.016Z
var datastore = new nedb({ filename: "test.db",  autoload: true, timestampData: true });
```

But the field `createdAt`, created by NeDB for each object, is not searchable by express-nedb-rest. When I do `http://127.0.0.1:8100/test?$filter=createdAt $regex 2017` the resulted answer is empty.

When I GET all the objects I can see `"createdAt":"2017-04-06T08:39:44.016Z"`.

But when I open my NeDB test.db file, surprise!, the date is stored like that:
`"createdAt":{"$$date":1491487725078}`

But `http://127.0.0.1:8100/test?$filter=createdAt $regex 14` doesn't work either (empty result).

This server side timestamp is very important for me (client side timestamps can be wrong). Any idea?

**Comment by bi-tm** [comment]
Hello Charles,
when you use the code  branch "dateHandling", you may filter dates. Please use operators $eq, $lt, $lt, ... The $regex operator will work with strings only.
Regards
Torsten

**Comment by CharlesNepote** [comment]
Hello Torsten,
Checked the "dateHandling" branch and tested $eq, $gt and $lt operators: all of them work fine. Thanks.
My best, Charles.

**Comment by bi-tm** [comment]
Hello Charles.
i published the new version as 1.2.0 on npm
Regards
Torsten

**Comment by CharlesNepote** [comment]
Good job!
I made a small pull request for small typos if you can have a look. I'm preparing another small one to improve the quality of the documentation.
Best, Charles.
